src/argilla/server/schemas/v1/datasets.py

```diff
@@ -79,3 +82,43 @@ class AnnotationCreate(BaseModel):
     title: str
     required: Optional[bool]
     settings: Union[TextAnnotationSettings, RatingAnnotationSettings] = Field(..., discriminator="type")
+
+
+class Response(BaseModel):
+    id: UUID
+    values: Dict[str, Any]
+    inserted_at: datetime
+    updated_at: datetime
+
+    class Config:
+        orm_mode = True
+
+
+class Record(BaseModel):
+    id: UUID
+    fields: Dict[str, Any]
+    external_id: Optional[str]
+    responses: List[Response]
+    inserted_at: datetime
+    updated_at: datetime
+
+    class Config:
+        orm_mode = True
```

Comment:
Since we have a records schema module, do we should move this definition to that module?